mirrord/layer/src/socket/ops.rs

```diff
@@ -819,8 +819,16 @@ pub(super) fn getaddrinfo(
     // Convert `service` into a port.
     let service = service.map_or(0, |s| s.parse().unwrap_or_default());
 
+    // Some apps (gRPC on Python) use `::` to listen on all interfaces, and usually that just means
+    // resolve on unspecified. So we just return that in IpV4 because we don't support ipv6.
+    let resolved_addr = if node == "::" {
+        vec![("::1".to_string(), IpAddr::V4(Ipv4Addr::UNSPECIFIED))]
```

Comment:
Maybe explain why `"::1"` if it's not supposed to be obvious by itself.

Reply:
Nice catch